exporter/fileexporter/file_exporter.go

```diff
@@ -112,15 +116,57 @@ func exportMessageAsBuffer(e *fileExporter, buf []byte) error {
 	if err := binary.Write(e.file, binary.BigEndian, data); err != nil {
 		return err
 	}
+
 	return nil
 }
 
+func startFlusher(e *fileExporter) {
+	if e.flushTicker != nil {
+		// flusher is already running.
+		return
+	}
+
+	if e.flushInterval < 1 {
+		// Interval should be 1 or larger.
+		return
+	}
```

Comment:
When `Start` calls `startFlusher`, I don't think there will be a situation where e.flushTicker != nil. As djaglowski said, we should determine whether to call the `startFlusher`  when the `Start` is called. So, can these two if statements be deleted?

Reply:
You have a point. I have changed it.